### DISCUSSION TITLE
Doesn't work on ArchLinux(Resolved)

### DISCUSSION THREAD

**Discussion by cupen** [fcitx/fcitx5#601] [General]
# Reproduce:

## Step 1: install
```bash
sudo pacman -Rs fcitx
sudo pacman -S fcitx5  fcitx5-rime fcitx5-qt fcitx5-gtk
```

## Step 2: setup `/etc/environment` and `~/.xprofile`
```bash
GTK_IM_MODULE=fcitx
QT_IM_MODULE=fcitx
XMODIFIERS=@im=fcitx
SDL_IM_MODULE=fcitx
GLFW_IM_MODULE=ibus
```

## Step 3:  run  `fcitx5` and  `fcitx-configtool`
![image](https://user-images.githubusercontent.com/1221615/190967495-6c57526b-0400-464e-87aa-588671c7f479.png)

## Step 4: type `ctrl-space` then it show me nothing but a `zh`.
![Peek 2022-09-19 15-26](https://user-images.githubusercontent.com/1221615/190968862-273623df-8e39-4404-a8d9-3c432ec014f7.gif)

# Resolved(已解决)
```
pacman -S fcitx5-chinese-addons
```
then restart fcitax5 and setup `Pinyin` in `fcitx5-configtool`.

![image](https://user-images.githubusercontent.com/1221615/190969951-48c60866-bfec-4a84-bf5f-02fbc756212e.png)
It works now.  Thank you guys.

**Comment by wengxt** [comment]
you’re adding the wrong item.
Please install fcitx5-chinese-addons, then add Pinyin instead.

**Reply by cupen** [reply]
> you’re adding the wrong item. Please install fcitx5-chinese-addons, then add Pinyin instead.

OH! YES! YES!  It works.  帅呆！ 谢谢你。

**Reply by sydddl** [reply]
谢谢
